quartz_solar_forecast/forecasts/v2.py

```diff
@@ -33,18 +37,17 @@ class TryolabsSolarPowerPredictor:
     """
     DATE_COLUMN = "date"
 
-    def _download_model(self, filename: str, file_id: str) -> None:
-        """
-        Download model from google drive and save it to the root of the repo.
-
-        Parameters
-        ----------
-        filename : str
-            The name of the model to be saved
-        file_id: 
-            Google id of the model file
-        """
-        gdown.download(f'https://drive.google.com/uc?id={file_id}', filename, quiet=False)
+    def _download_model(self, filename: str, repo_id: str, file_path: str) -> str:
+        # Use the project directory instead of the user's home directory
```

Comment:
And here

Reply:
done @peterdudfield , any other changes?